README.md

```diff
@@ -1,85 +1,167 @@
 # React PropTypes
 
 ## Objectives
+1. Describe why propTypes are a handy tool
+2. Describe how to use `PropTypes` declarations
+3. Explain when to use `isRequired`
+4. Explain when to use `PropTypes.object` instead of `PropTypes.shape()`
 
-1. Describe how to use `PropTypes` declarations
-2. Describe why propTypes are a handy tool
-3. Explain when to use `PropTypes.object` instead of `PropTypes.shape()`
-4. Explain when to use `isRequired`
-
-## Overview
-
-This lesson should be reasonably straightforward, but it's important to give
-students as much practice as possible. Frame it as some sort of story — maybe
-we're at the DMV and we have to process documents, and they need to be of
-specific types?
-
-Anyway, I think students with a basic understanding of JavaScript will find most
-PropTypes intuitive. The difference between `object` and `shape()` is something
-that I happen to have strong opinions about though — feel free to disregard the
-following and change the third objective if this seems too heavy-handed.
-
-`PropTypes.object` is great for when we know we want some kind of object, but
-we don't (yet) care what's in it. This is for wrapper or container components
-that don't need to validate anything.
-
-But usually, we'll want `PropTypes.shape()`, which is better for when we know
-that we're going to be rendering specific parts of an object. We only need to
-account for the specific parts that we're using the component we're in, but it
-_really_ helps make components easy to understand.
-
-Let's say we have an address component (DMVs need those), and we want to pull
-the address information out of a `address` object. `shape()` is super handy:
-
-```javascript
-const React = require('react')
-const PropTypes = React.PropTypes
-
-class Address extends React.Component {
-  // as with defaultProps, we'll need to include
-  // the correct transform for this to work
-  propTypes = {
-    address: PropTypes.shape({
-      streetAddress: PropTypes.string.isRequired,
-      streetAddress2: PropTypes.string,
-      city: PropTypes.string.isRequired,
-      state: PropTypes.string.isRequired,
-      zipCode: PropTypes.string.isRequired
-    }
+## What are PropTypes?
+PropTypes is a way for us to enforce that components receive the right props in the right form. Basically, it's kind of
+a developer-oriented validation: it catches errors when you pass the wrong stuff into a component, and warns you about
+it. These warnings only show up in development versions of React — this stuff isn't checked in production versions
+because of the overhead (and throwing the errors in production would be too little too late anyway).
```

Comment:
This is definitely not a must, but it might be worth mentioning that setting PropTypes can help with code reuse and modularity. Because the PropTypes are declared the developer who didn't write the module but wants to reuse it will more quickly understand what the module is what and is required to use it.